### PR TITLE
Feature 5: Listing Detail Page

### PR DESCRIPTION
**Pull request by lukecheseldine**
- reworked how ids work for Listing class. implemented an atomic counter from 0
- created ListingDetailPage, dependent upon other code and currentUser for final implementation

thoughts?

### CODE REVIEW

**Review by liu-samuel — DISMISSED** [review]
lgtm

File: src/main/java/forms/CreateListingForm.java [hunk 1]
```diff
@@ -13,27 +13,25 @@ public class CreateListingForm extends Form {
...
         this.description = description;
         this.images = images;
+        this.id = id;
     }
 
 
     public void submitListing() {
         // This function will be called on the "submit" button click
-        // This will add a listing to the databse using the database interactor method.
+        // This will add a listing to the database using the database interactor method.
```
Comment (MarkianV) [inline src/main/java/forms/CreateListingForm.java:32]: Add //TODO somewhere here

**Review by MarkianV — APPROVED** [review]
overall good, if you place the TODO it should be perfect for now.

**Review by danielqiuu — APPROVED** [review]
Thanks for reworking the id's

File: src/main/java/forms/CreateListingForm.java (hunk 1 above)
Comment (lukecheseldine) [reply src/main/java/forms/CreateListingForm.java:32]: done!

**Comment by lukecheseldine** [comment]
i just added the todo that you wanted mark, no other changes. just need y'all to reapprove :)

**Review by ethanrstark — APPROVED** [review]

**Review by claregillis — APPROVED** [review]
lloks good!

**Review by danielqiuu — APPROVED** [review]

**Pull request merged**